### ISSUE TITLE
affinity.serif.com

### ISSUE THREAD

**Issue by Blitheness** [RooneyMcNibNug/pihole-stuff#50]
This is a graphic design software website blocked by the SNAFU list. I can understand the email subdomain being blocked but I'm not sure why their main website is.

**Comment by ThreepwoodLeBrush** [comment]
I second this report, affinity.serif.com should not be blocked.

**Comment by RooneyMcNibNug** [comment]
Thank you for this issue! I think that this was enumerated by mistake, to be honest. I've removed the entire domain enumeration from the blocklist, as even the `email` subdomain seems benign.

The following has been removed now:

```
email.affinity.serif.com
cdn.affinity.serif.com
press.affinity.serif.com
forum.affinity.serif.com
affinity.serif.com
```

This has been completed in the following commit: https://github.com/RooneyMcNibNug/pihole-stuff/commit/57d8fcc10c9d6bca64f0be479d9a9adedbfc8788

Let me know if you have any other issues at all.

Safe surfing,
_Rooney_
